CHANGELOG.md

```diff
@@ -197,6 +197,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated passing `process_position` to the `Trainer` constructor in favor of adding the `ProgressBar` callback with `process_position` directly to the list of callbacks ([#9222](https://github.com/PyTorchLightning/pytorch-lightning/pull/9222))
 
 
+- Deprecated `LightningLoggerBase.close`, `LoggerCollection.close` in favor of `LightningLoggerBase.save`, `LoggerCollection.save` ([#9422](https://github.com/PyTorchLightning/pytorch-lightning/pull/9422))
```

Comment:
why's this duplicated? 

also @awaelchli @edward-io is there consensus on whether we should recommend the usage of `save` vs. `finalize` here? 
```suggestion
- Deprecated `LightningLoggerBase.close` in favor of `LightningLoggerBase.save` ([#9422](https://github.com/PyTorchLightning/pytorch-lightning/pull/9422))
```

Reply:
- "duplicated": `LightningLoggerBase` and `LoggerCollection` ?
- yea I was wondering this too. I picked `save` here since `LightningLoggerBase.close` just calls `self.save` for identical functionality (v.s. `finalize` could've been re-implemented differently)

Reply:
> "duplicated": LightningLoggerBase and LoggerCollection ?

ah sorry, didn't read carefully!

Reply:
normally, saving and closing would be different concepts. So I would favor finalize as the replacement. Unfortunately I don't know the original intentions of the authors behind these methods.

Reply:
Agreed that finalize would be the closest thing to close.